corehq/apps/userreports/expressions/utils.py

```diff
@@ -17,7 +18,11 @@ def safe_range(start, *args):
 
 
 SAFE_OPERATORS = copy.copy(DEFAULT_OPERATORS)
-SAFE_OPERATORS[ast.Pow] = safe_pow_fn  # don't allow power operations
+SAFE_OPERATORS.update({
+    ast.Pow: safe_pow_fn,  # don't allow power operations
+    ast.Or: op.or_,  # Allows coalesce
+    ast.And: op.and_,  # Can't have "or" without "and". Allows ternary
```

Comment:
I left out `op.not_` because it returns a boolean, which is not an allowed data type.